lib/triggers/csgostatTrigger.js

```diff
@@ -75,6 +75,9 @@ CSGOStatTrigger.prototype._respond = function(toId, userId, message) {
 		});
 		return true;
 	}
+	else if(query && params.length == 1) {
+		this._sendMessageAfterDelay(toId, 'Error reading command, use: ' + this.options.command + ' <steamID64>');
+		return false;
 	else {
```

Comment:
You'll need to add a closing curly brace, right? This will throw an error for something like `Unexpected "else"`.